javascript/builtins/Array.json

```
@@ -1929,6 +2037,16 @@
                 "chrome_android": {
                   "version_added": "25"
                 },
+                "deno": [
+                  {
+                    "version_added": "1.8"
+                  },
+                  {
+                    "version_added": "1.0",
+                    "partial_implementation": true,
+                    "notes": "Before version 1.8, only the locale data for <code>en-US</code> is available."
```

There's a weird subtlety to keep in mind when there are multiple support statements: if two support statements overlap, they need to make sense together. As written, I'd read it as:

- From version 1.8, supported with no limitations.
- From version 1.0, partially supported (with a limitation, noted).

These don't really make sense together (they could if, for example, one had an alternative name and the other did not). I suspect this is what you really meant:

- From version 1.8, supported with no limitations.
- From version 1.0 to 1.8, partially supported (with a limitation, noted).

And the source changes to make that so:

```suggestion
                    "version_added": "1.0",
                    "version_removed": "1.8"
                    "partial_implementation": true,
                    "notes": "Only the locale data for <code>en-US</code> is available."
```